bin/eks-create-sc.sh

```diff
@@ -6,15 +6,17 @@
 # You must have the gcloud command installed and access to a GCP project.
 # See https://cloud.google.com/container-engine/docs/quickstart
 
+
+source "${BASH_SOURCE%/*}/../etc/eks-env.cfg"
```

Comment:
I don't see any variables from eks-env.cfg being used in this file.  Best to add this line when required.